--- nsswitch.conf	2022-05-28 23:20:39.122400529 +0100
+++ nsswitch.conf.new	2022-02-20 21:58:22.000000000 +0000
@@ -3,12 +3,11 @@
 
 passwd: files systemd
 group: files [SUCCESS=merge] systemd
-shadow: files systemd
-gshadow: files systemd
+shadow: files
 
 publickey: files
 
-hosts: mymachines resolve [!UNAVAIL=return] files myhostname dns
+hosts: mymachines myhostname mdns_minimal [NOTFOUND=return] resolve [!UNAVAIL=return] files myhostname dns wins
 networks: files
 
 protocols: files
